### ISSUE TITLE
set mtf_image programmatically

### ISSUE THREAD

**Issue by irsalshabirin** [florent37/MaterialTextField#16]
please add method to set icon/image programmatically,
thanks before 👍

**Comment by tranqui341** [comment]
I agree with disabling the reduce() on touch when EditText contains text. I also have my implementation reduce() when the EditText loses focus when it's empty... looks really good that way.

![mtf](https://user-images.githubusercontent.com/6187169/38463546-79cdcf80-3ab2-11e8-96b1-a610fdc6e2d9.gif)
